e2e/playwright/st_status_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from playwright.sync_api import Page, expect
+
+from conftest import ImageCompareFunction
+
+
+def test_status_container_rendering(
+    themed_app: Page, assert_snapshot: ImageCompareFunction
+):
+    """Test that st.status renders correctly via screenshots."""
+    status_containers = themed_app.get_by_test_id("stExpander")
+    expect(status_containers).to_have_count(9)
+
+    # Don't check screenshot for first element,
+    # since we cannot reliable screenshot test the spinner icon.
+    for i, element in enumerate(status_containers.all()[1:]):
+        assert_snapshot(element, name=f"status-{i}")
```

Comment:
Is it possible to make these test names more specific rather than some index? When I read `status-0`, it's hard to understand what `status-0` is and then one will have to go and read the test / e2e script.

Reply:
Change this to use more descriptive names 👍